external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -139,8 +139,8 @@ public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
 
         @Override
         public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
-            LOG.info("Partitions reassignment. [consumer-group={}, consumer={}, topic-partitions={}]",
-                    kafkaSpoutConfig.getConsumerGroupId(), kafkaConsumer, partitions);
+            LOG.info("Partitions reassignment. [task-ID={}, consumer-group={}, consumer={}, topic-partitions={}]",
```

Comment:
Minor thing, just pointing it out, though it's probably fine this way:  the "task ID" is referred to in 3 different styles in these log lines through this PR:
* `taskId`
* `Task-ID`
* `task-ID`

I think each is consistent within their respective log lines.  Just wondering if there's any value to it being consistent across them.  Also wonder if there's any existing convention in other log lines in the code.